cmd/root.go

```diff
@@ -59,7 +59,10 @@ func Execute(version string) {
 		os.Exit(1)
 	}
 
-	_ = rootCmd.Execute()
+	err = rootCmd.Execute()
+	if err != nil {
+		os.Exit(1)
+	}
```

Comment:
I have a problem with this code.

There should be no os.Exit in code outside main.go. 

I would like to suggest you to update the signature of execute to return an error. And to have this if err != nil os.Exit(1) in main.go

It's not related to this PR, the problem was already there. But I'm discovering it now, so I report it now